wp-downloadmanager.php

```diff
@@ -1576,3 +1576,57 @@ function downloadmanager_activate() {
 
 	flush_rewrite_rules();
 }
+
+function downloads_generate_user_roles_select( $permission = array(), $mode = 'create' ) {
+
+	$wp_roles = wp_roles();
+	echo 'Administrators will have access to all downloads, that can\'t be protected. <br />';
+	$select = '<select name="file_permission[]" multiple>';
+	$options = '';
+	if ( $mode == 'edit' ) {
+		$permission = explode( '+', $permission );
```

Comment:
Actually, any reason we use + instead of ","? I would prefer it to be "," rather than "+" as it is more readable and doesn't need to be urlencoded